blobstore/src/main/java/org/jclouds/blobstore/TransientStorageStrategy.java

```diff
@@ -193,7 +195,14 @@ public String putBlob(final String containerName, final Blob blob, BlobAccess ac
          Closeables2.closeQuietly(input);
       }
 
-      Blob newBlob = createUpdatedCopyOfBlobInContainer(containerName, blob, payload, actualHashCode);
+      String eTag = null;
+      if (blob.getMetadata() != null) {
+         eTag = blob.getMetadata().getETag();
+      }
+      if (eTag == null || !MPU_ETAG_FORMAT.matcher(eTag).matches()) {
```

Comment:
Why does this need to sanity check the format?  Is there a danger to allowing an arbitrary input?

Reply:
That's fair. Removed the format check.